test/adagrams.test.js

```diff
@@ -120,7 +120,10 @@ describe("Adagrams", () => {
     });
 
     it("returns a score of 0 if given an empty input", () => {
-      throw "Complete test";
+      // throw "Complete test";
+      const word = ""
+      const score = scoreWord(word);
+      expect(score).toEqual(0);
```

Comment:
This works but a DRY-er version of this code would be to use the `expectScores` function.